crates/networking/p2p/tx_broadcaster.rs

```diff
@@ -229,7 +229,7 @@ impl TxBroadcaster {
                 transactions: txs_to_send,
             });
             connection.outgoing_message(txs_message).await.unwrap_or_else(|err| {
-                error!(peer_id = %format!("{:#x}", peer_id), err = ?err, "Failed to send transactions");
+                debug!(peer_id = %format!("{:#x}", peer_id), err = ?err, "Failed to send transactions");
```

Comment:
I think it's ok to have this on `error`: If it fails the broadcaster is broken and user should be aware.